### ISSUE TITLE
Write simple note plugin

### ISSUE THREAD

**Issue by mmazer** [mmazer/config.nvim#14]
- follow note links in [[]]
- See https://github.com/mmazer/config.nvim/commit/022732248970dc319839ae2a7cbdf73df2faaaa1 for previous `zettel` plugin

**Comment by mmazer** [comment]
See https://www.reddit.com/r/vim/comments/smwje2/copy_inside_of_brackets_to_variable/ for selecting text between `[]`

**Comment by mmazer** [comment]
Or use a macro `vi[gf` wrapped in function.

**Comment by mmazer** [comment]
To select text within `[[]]` use something like:
```vim
function! lh#visual#selection()
  try
    let a_save = @a
    normal! gv"ay
    return @a
  finally
    let @a = a_save
  endtry
endfunction
```

**Comment by mmazer** [comment]
Fixed in eb28d01879dadcf0f00853a935b72109bea2f1a4
